packages/eas-cli/src/credentials/ios/appstore/provisioningProfileAdhoc.ts

```diff
@@ -1,9 +1,208 @@
+import { Device, Profile, ProfileState, ProfileType } from '@expo/apple-utils';
 import ora from 'ora';
 
-import { ProvisioningProfile } from './Credentials.types';
 import { AuthCtx } from './authenticate';
+import { getBundleIdForIdentifierAsync, getProfilesForBundleIdAsync } from './bundleId';
+import { ProvisioningProfile } from './Credentials.types';
+import { getDistributionCertificateAync } from './distributionCertificate';
+import { USE_APPLE_UTILS } from './experimental';
 import { runActionAsync, travelingFastlane } from './fastlane';
 
+interface ProfileResults {
+  provisioningProfileUpdateTimestamp?: number;
```

Comment:
i originally had `provisioningProfileUpdateTimestamp` and `provisioningProfileCreateTimestamp` so i could call out to the ruby scripts and find out whether we 1.) created, 2.) updated or 3.) used an existing profile without updating. In hindsight it'd have been simpler if we just passed a `didUpdate` or `didCreate` property instead.

If you could change these to `didUpdate` and `didCreate` instead, that'd be amazing.